test/BUILD.gn

```diff
@@ -710,6 +711,12 @@ test("brave_browser_tests") {
     ]
   }
 
+  if (enable_playlist) {
+     sources += [
+      "//brave/browser/playlist/playlist_browsertest.cc",
```

Comment:
test should go in the target they belong to

Reply:
also this is definitely missing deps since playlist is new

Reply:
Done.  `"//brave/browser/playlist/test:browser_tests` is introduced.